app/views/personas/index.html.erb

```diff
@@ -11,14 +11,14 @@
     <%= govuk_table do |table|
       table.with_colgroup do |colgroup|
         colgroup.with_col(span: 1)
-        colgroup.with_col(span: 6)
+        colgroup.with_col(span: 7)
         colgroup.with_col(span: 1)
       end
 
       table.with_head do |head|
         head.with_row do |row|
           row.with_cell(scope: false)
-          row.with_cell(header: true, text: "Permissions", colspan: 5)
+          row.with_cell(header: true, text: "Permissions", colspan: 6)
```

Comment:
```suggestion
          row.with_cell(header: true, text: "Permissions", colspan: 7)
```

I actually think this needs to be 7 because there are 7 permission columns (I think the value of 5 before was incorrect).